src/authenticated.js

```diff
@@ -1,17 +1,19 @@
 (function() {
     const onlyLoggedOut = document.querySelector('.only-loggedout'),
         onlyLogged = document.querySelector('.only-logged'),
-        onlyLoggedProfile = document.querySelector('.only-logged-profile');
+        onlyLoggedProfile = document.querySelector('.only-logged-profile'),
+        authReq = new XMLHttpRequest();
 
     // See if user is authenticated
-    fetch('/authenticated')
-    .then((resp) => resp.json())
-    .then((data) => {
+    authReq.open('GET', '/authenticated', true);
```

Comment:
I can add a polyfill there is a very small one arround and I'll integrate it in the site. 

I'm refactoring the js to divide into modules and I'll do that too